tests/pkg_winc1500_netdev/Makefile

```diff
@@ -0,0 +1,29 @@
+APPLICATION = driver_winc1500_netdev
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := msb-430 msb-430h nucleo32-f031 nucleo32-f042 \
+                             nucleo32-l031 nucleo-l053 stm32f0discovery telosb \
+                             weio z1
+
+USEPKG += winc1500
+
+## Comment out when you don't need GNRC. Basic shell commands
```

Comment:
`# Comment out when you don't need GNRC.` part looks like a copy paste, should be removed